src/main/java/org/openstreetmap/atlas/checks/validation/points/ConnectivityCheck.java

```diff
@@ -64,6 +65,8 @@ public ConnectivityCheck(final Configuration configuration)
         this.denylistedHighwaysTaggableFilter = TaggableFilter
                 .forDefinition(this.configurationValue(configuration, "denylisted.highway.filter",
                         DEFAULT_DENYLISTED_HIGHWAYS_TAG_FILTER));
+        this.checkedHighwayValues = this.configurationValue(configuration, "checked.highway.values",
```

Comment:
Could you add a note in the md file for this check describing this configurable?